.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: node_js
 node_js:
-  - 0.8
-  - 0.9
+  - 4
```

Comment:
Node.js 4 is officially end-of-life, although maybe you still want to support it for a while anyways?

It would be nice to add the `"engines"` field to the `package.json` 🙌